test/unit/format_capture_helper_test.rb

```diff
@@ -0,0 +1,87 @@
+require_relative "../test_helper"
+
+module MockCaptureHelper
+  def content_for(_name, content, _options, &_block)
+    content
+  end
+
+  def capture(&block)
+    block.call
+  end
+
+  def render(*_args, &block)
+    "<g>" + block.call + "</g>"
+  end
+end
+
+class MockClass
+  include MockCaptureHelper
```

Comment:
Not quite sure why these methods are imported in since they don't seem to be used outside this, I think you'd have same effect defining these as methods of the class.

Reply:
Now directly written the class definition.